README.md

```diff
@@ -114,3 +114,45 @@ The interesting fields for this use case are:
   the SSO user ID to be sent to Segment
 - `status.compliantUsername` - Contains the username used in the cluster audit
   log.
+
+## Setting up a containerized Splunk instance
+- Either podman or docker can be user
+- running the following command will deploy a Splunk enterprise container:  
+- `podman pull docker.io/splunk/splunk:latest`
+  `podman run -p 8000:8000 -e "SPLUNK_PASSWORD=AcceptablePassword1" -e "SPLUNK_START_ARGS=--accept-license" -it --name splunk_container docker.io/splunk/splunk:latest`
+- More info about the specific inputs of the command can be found [here|https://docs.splunk.com/Documentation/Splunk/9.0.4/Installation/DeployandrunSplunkEnterpriseinsideDockercontainers]
+
+
+## Setting up a containerized Splunk instance
+
+To set up a containerized Splunk instance, you can use either podman or docker. Follow these steps:
+
+1. Pull the latest Splunk Enterprise container image:
+   ```
+   podman pull docker.io/splunk/splunk:latest
+   ```
+
+2. Run the container with the desired configuration:
+   ```
+   podman run -p 8000:8000 -e "SPLUNK_PASSWORD=YourPassword" -e "SPLUNK_START_ARGS=--accept-license" -it --name splunk_container docker.io/splunk/splunk:latest
+   ```
+   This command maps port 8000 from the container to the host machine, sets the Splunk password to meet the required complexity, and accepts the Splunk license.
+
+   For more information about the specific command inputs and options, refer to the [Splunk documentation](https://docs.splunk.com/Documentation/Splunk/9.0.4/Installation/DeployandrunSplunkEnterpriseinsideDockercontainers).
+
+
+3. Once the container is running, you can log in to the Splunk instance using the username `admin` and the password you set up.
+
+
+4. To access the Splunk UI, open a web browser on the host and navigate to `localhost:8000`.
+
+
+* If you prefer to use the Splunk REST API, follow these steps:
+   - Get inside the running container using the command:
```

Comment:
Why do I have to do this? Pleas also expose port 8089 so we can do the curl calls directly from the host.

Also please add a link to the API docs somewhere.

Reply:
done